src/main/scala/cromwell/engine/backend/jes/authentication/JesAuthentication.scala

```diff
@@ -20,7 +35,12 @@ object ProductionJesConnection {
  */
 trait JesAuthentication { self: JesConnection =>
 
-  def authenticated[A](f: JesInterface => A) = f(jesConnection)
+  def authenticatedAsCromwell[A](f: JesInterface => A) = f(jesCromwellConnection)
```

Comment:
I think I'd prefer something like `authenticateAsCromwell` and `authenticateAsUser` - the ordering sounds more natural to me. OTOH I always did terribly in grammar class and such when i was young